MSAL/src/native_auth/network/responses/validator/reset_password/MSALNativeAuthResetPasswordResponseValidator.swift

```diff
@@ -62,7 +62,7 @@ final class MSALNativeAuthResetPasswordResponseValidator: MSALNativeAuthResetPas
                            context: context,
                            format: "resetpassword/start returned success with unexpected response body")
 
-            return .unexpectedError
+            return .unexpectedError(message: nil)
```

Comment:
Can you create a new error message that explain why this error happened?
Something like: "Unexpected response body received"

Reply:
Added the message "Unexpected response body received" everywhere where a field is missing or the body cannot be decoded